### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by amymareerobinson**
Please check the formatting, naming, and logic, specifically for emails.py @JulieRoder

### CODE REVIEW

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Student Name: Amy Robinson
+Program - Hex Colours
+"""
+
+HEX_COLOURS = {"blueviolet": "#8a2be2", "coral": "#ff7f50",
+               "firebrick": "#b22222", "forestgreen": "#228b22",
+               "hotpink": "#ff69b4", "brown": "#a52a2a",
+               "chocolate": "#d2691e", "floralwhite": "#fffaf0",
+               "navyblue": "#000080", "light": "#eedd82"}
+
```
Comment (JulieRoder) [inline prac_05/hex_colours.py:12]: How would the user know what colours are available? maybe printing a list of the dictionary keys.

File: prac_05/state_names.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                "ACT": "Australian Capital Territory",
+                "VIC": "Victoria", "TAS": "Tasmania"}
+
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ").upper()
+
+while state_code != "":
+    if state_code in CODE_TO_NAME:
+        print(state_code, "is", CODE_TO_NAME[state_code])
```
Comment (JulieRoder) [inline prac_05/state_names.py:22]: Do you think that print statements should have consistent formats?

File: prac_05/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+word_count_dict = {}
+
+text = input("Text: ")
+words = sorted(text.split())
+
+for word in words:
+    word_count_dict[word] = word_count_dict.get(word, 0) + 1
+
+longest_word = max((len(word) for word in words))
```
Comment (JulieRoder) [inline prac_05/word_occurrences.py:15]: I did a test and the double brackets for max are not needed.

File: prac_05/emails.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+        if verify_name == 'y' or verify_name == "":
+            email_dict[email] = name
+
+        elif verify_name == 'n' or verify_name == 'no':
+            name = input("Name: ").title()
+            email_dict[email] = name
+
+        else:
+            print("Invalid verification")
```
Comment (JulieRoder) [inline prac_05/emails.py:25]: What follows? Is the name not entered? how does the user know what has happened? maybe add "name could not be entered" after invalid message, or loop it to ask for verification again.

File: prac_05/hex_colours.py (hunk 1 above)
Comment (amymareerobinson) [reply prac_05/hex_colours.py:12]: Added, Thank you.

File: prac_05/state_names.py (hunk 2 above)
Comment (amymareerobinson) [reply prac_05/state_names.py:22]: Changed, Thank you.

File: prac_05/word_occurrences.py (hunk 3 above)
Comment (amymareerobinson) [reply prac_05/word_occurrences.py:15]: Deleted, Thank you.

File: prac_05/emails.py (hunk 4 above)
Comment (amymareerobinson) [reply prac_05/emails.py:25]: Interesting, really good point and recommendation. Thank you, I changed it.

**Pull request merged**